perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1281,6 +1281,11 @@ sub user_edit : Path('user_edit') : Args(1) {
             $user->from_body( undef );
         }
 
+        # Some cobrands (e.g. Oxfordshire) have their own custom fields on User
```

Comment:
Could this be in a cobrand function? (Probably one not bothering to be in Default)